source/_pages/blog.twig

```diff
@@ -22,10 +22,12 @@ use:
             <h2 class="sidebar__title">Recent posts:</h2>
             <ul class="sidebar__list">
                 {% for recent in data.posts %}
+                    {% if loop.index < 8 %}
```

Comment:
Maybe you can `{% for recent in data.posts|slice(0,8) %}`?

Reply:
done